### PR TITLE
Add some basic tests for the Voigt module

### PR DESCRIPTION
**Pull request by smokestacklightnin**
### :pencil: Description

**Type:** :vertical_traffic_light: `testing` 

This pull request adds a few basic tests to the [`opacities.voigt`](https://github.com/tardis-sn/stardis/blob/main/stardis/opacities/voigt.py) module. The tests are parametrized to test several input values and also include tests to raise errors when a division by zero is attempted. 

### :pushpin: Resources

[How to parametrize pytest fixtures and test functions](https://docs.pytest.org/en/7.3.x/how-to/parametrize.html)

[Examples for parametrized test functions](https://docs.pytest.org/en/7.3.x/example/parametrize.html#paramexamples)

### :vertical_traffic_light: Testing

How did you test these changes?

- [X] Testing pipeline
- [X] Tested locally

### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

### CODE REVIEW

**Review by sonachitchyan — APPROVED** [review]

**Review by smokestacklightnin — COMMENTED** [review]
@jaladh-singhal is something like this what you had in mind? i know it's a few more than you suggested.

 @jvshields

**Review by jvshields — APPROVED** [review]

File: stardis/opacities/tests/test_voigt.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+)
+@pytest.mark.parametrize(
+    "voigt_profile_division_by_zero_input_gamma",
+    test_voigt_profile_division_by_zero_test_values,
+)
+def test_voigt_profile_division_by_zero(
+    voigt_profile_division_by_zero_input_delta_nu,
+    voigt_profile_division_by_zero_input_gamma,
+):
+    from stardis.opacities.voigt import voigt_profile
```
[the remark this answers is not in the document]
Comment (smokestacklightnin) [reply stardis/opacities/tests/test_voigt.py]: I just [moved the imports to the beginning of the file](https://github.com/tardis-sn/stardis/pull/94/commits/0dce29b040403def50353defb65087b7531bf95c)

**Review by sonachitchyan — APPROVED** [review]

**Review by jvshields — APPROVED** [review]
